lib/ephemeral/client.rb

```diff
@@ -23,6 +23,12 @@ def build(image, repo, build_type)
         })
     end
 
+    def getLogs(id)
+      resource = "builds/#{id}/logs"
+
+      get(resource, {})
```

Comment:
Not the comment on made on `def get`, this you will be able to update to `get(resource)`